sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -242,6 +242,35 @@ object AnalysisContext {
   }
 }
 
+object Analyzer {
+  // List of configurations that should be passed on when resolving views and User-Defined Table
+  // Valued functions
```

Comment:
SQL UDF supports both scalar function and table function. Let's just say `user-defined SQL functions`

Reply:
see it. Added